### PR TITLE
Kim mb 705 office make model weight ticket review

### PR DESCRIPTION
**Pull request by kimallen**
## Description

On the office side, the user should see make and model fields for the car or car and trailer weight tickets. in the document viewer

## Reviewer Notes

2 other fixes: 
the progear type had the "Vehicle nickname" label, changed it to progear label
The font size of the weight-ticket-specific field labels was extra large- fixed to match the other fields
Fixed capitalization inconsistencies for field labels (as `Field label`)

The ticket mentions an issue with the value of nickname showing up in the notes field.  I was not able to recreate the issue.

Note: I added tests for the `DocumentDetailDisplay`.  Test for `DocumentDetailEdit` will come in a future PR.

## Setup

`make server_run`
`make client_run`
Log in as office user
Choose Payment Requested Queue,  REQPAY move.
Click on a weight ticket in the documents list.
Edit and check the different fields that appear based on weight ticket type.

## Code Review Verification Steps

* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-705) for this change

## Screenshots

image
image

### CODE REVIEW

File: src/scenes/Office/DocumentViewer/DocumentDetailPanel.jsx [hunk 1]
```diff
@@ -129,30 +161,36 @@ const DocumentDetailEdit = ({ formValues, moveDocSchema }) => {
...
+              </div>
+              {isWeightTicketTypeBoxTruck && (
+                <SwaggerField fieldName="vehicle_nickname" swagger={moveDocSchema} required />
+              )}
+              {isWeightTicketTypeProGear && (
                 <SwaggerField
-                  className="short-field"
-                  fieldName="weight_ticket_set_type"
+                  fieldName="vehicle_nickname"
+                  title="Pro-gear type (ex. 'My Pro-gear', 'Spouse Pro-Gear', 'Both')"
```
Comment (lynzt) [inline src/scenes/Office/DocumentViewer/DocumentDetailPanel.jsx]: standardize caps...

**Comment by kimallen** [comment]
Changed the caps issues.  Re: the tests, I think the component we'd want to test is the `DocumentDetailPanel`.  There are currently no tests at all for this one.  I'm wondering if it's because it opens up into a new window- is that testable?

**Comment by lynzt** [comment]
This looks good to me but would be curious what @donaldthai thinks...

**Comment by donaldthai** [comment]
Code looks good. There is a console error on that document view page that keeps popping up:
```
index.js:1 Warning: Cannot update a component from inside the function body of a different component.
    in Form(Wrapper) (created by ConnectFunction)
    in ConnectFunction (created by Connect(Form(Wrapper)))
    in Connect(Form(Wrapper)) (created by ReduxForm)
    in ReduxForm (created by Context.Consumer)
    in Hoc (created by ReduxForm)
    in ReduxForm (created by ConnectFunction)
    in ConnectFunction (at DocumentViewer/index.jsx:181)
    in div (created by TabPanel)
    in TabPanel (at DocumentViewer/index.jsx:180)
    in div (created by UncontrolledTabs)
    in UncontrolledTabs (created by Tabs)
    in Tabs (at DocumentViewer/index.jsx:159)
    in div (at DocumentViewer/index.jsx:158)
    in div (at DocumentViewer/index.jsx:154)
    in div (at DocumentViewer/index.jsx:137)
    in div (at DocumentViewer/index.jsx:136)
    in DocumentViewer (created by ConnectFunction)
    in ConnectFunction (at Office/index.jsx:46)
    in main (at Office/index.jsx:45)
    in Suspense (at Office/index.jsx:43)
    in RenderWithOrWithoutHeader (at Office/index.jsx:139)
    in Suspense (at Office/index.jsx:138)
    in component (created by Context.Consumer)
    in Route (at PrivateRoute.jsx:14)
    in PrivateRouteContainer (created by ConnectFunction)
    in ConnectFunction (at Office/index.jsx:135)
    in Switch (at Office/index.jsx:92)
    in ConditionalWrap (at Office/index.jsx:81)
    in div (at Office/index.jsx:79)
    in Router (created by ConnectedRouter)
    in ConnectedRouter (created by Context.Consumer)
    in ConnectedRouterWithContext (created by ConnectFunction)
    in ConnectFunction (at Office/index.jsx:78)
    in OfficeWrapper (created by ConnectFunction)
    in ConnectFunction (at AppContext/index.jsx:27)
    in ContextualComponent (created by LoadableComponent)
    in LoadableComponent (at App/index.jsx:46)
    in Provider (at App/index.jsx:44)
    in App (at src/index.js:11)
```

**Comment by kimallen** [comment]
Turns out the error exists in master as well.  I'll create a bug ticket for it,.

**Pull request merged**